frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css

```diff
@@ -0,0 +1,81 @@
+@import "../../config/colors.css";
+@import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
+
+#forgotPasswordBox {
+    width: 610px;
+    height: 420px;
+    margin: auto;
+    left: 515px;
+    top: 407px;
+
+    box-sizing: border-box;
+
+    background: var(--white-gray);
+    margin: auto;
+    margin-top: 50vh;
+    transform: translateY(-50%);
+    box-shadow: 0px 8px 21px var(--box-shadow);
+    padding: 55px 85px 68px 85px;
+}
+
+#forgotPasswordText {
+    font-family: Rubik;
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 40px;
+    margin: 0px 0px 34px 0px;
+    text-align: left;
+
+    color: var(--black);
```

Comment:
nit: the color on Figma is an off black (#011338) not sure how much this matters